### DISCUSSION TITLE
DataFusion meetup in Midwest USA (Detroit, Chicago, etc)

### DISCUSSION THREAD

**Discussion by timsaucer** [apache/datafusion#11568] [Ideas]
Referencing @alamb 's posts for other geographies:

> After our first meetup, (see https://github.com/apache/datafusion/discussions/8152 / https://github.com/apache/datafusion/discussions/8522) I have heard there may be interest in other meetups in different geographies.
> 
> I would personally love to support such meetups (will promote, try to attend, can help find sponsors, etc) as much as possible but I don't have the bandwidth to actually organize them.
> 
> Anyone who is interested in attending or helping to organize please leave a comment
> 
> Other meetup discussions:
> https://github.com/apache/datafusion/discussions/10347

Is there any interest in a meetup in the Midwest USA? Potential areas could be Detroit, Chicago, Indianapolis.

**Comment by alamb** [comment]
I would be interested in attending (I am often in the Milwaukee area for personal reasons) and would be very happy to help advertise / promote such a gathering. I think the key ingredients for a successful meetup are:
1. A base of enough people locally who could/would attend
2. someone to organize (find a venue, wrangle speakers, etc)

**Reply by adriangb** [reply]
I am in Chicago! I'd be happy to help host.

For venues I can put something together for a smaller group and if larger I can see if someone with an office can host.
I'll also reach out to the Chicago Python and Data Happy Hour communities.

Andrew do you have any preference on dates? The Chicago Data Happy Hour is a fun event, Hannes was there a couple months ago. I think they do it the 3rd Monday of the month. We could shoot for October if that's not too soon?

**Reply by alamb** [reply]
Oct and Nov are already pretty booked -- any chance you could help organize something the week of Dec 16? I may be in the area anyways. I know it is a bit chilly that time of year, but what better to warm things up than a DataFusion meetup!

**Reply by adriangb** [reply]
December 16th works for me! Data Happy Hour should be that day as well.

Shall I make a Luma link and start looking for speakers?

**Reply by Michael-J-Ward** [reply]
@adriangb - I'm also in Chicago and would love to lend a hand to help organize an event here.

**Reply by alamb** [reply]
Any chance we can do something Wed Dec 18? @adriangb  or @Michael-J-Ward  can you help me find some venue? I can help organize the logistics, speakers, content, etc

**Reply by adriangb** [reply]
Yeah that works for me. I don't have an office to volunteer but I'll start asking around!

**Reply by adriangb** [reply]
A couple of options I've come up with:
- [Harold Washington Library](https://maps.app.goo.gl/pdbuXeTQPPoNYqYi6) is a beautiful library downtown that has a wide range of meeting spaces available. Large conference rooms (40 people) are ~$250 for a day.
- [UIC](https://maps.app.goo.gl/D49QgqDXSKZGoHAK7) has a [databases group](https://www.cs.uic.edu/~bglavic/dbgroup/) that I reached out to see if the university might be able to host it and to see if they'd be interested in submitting a talk or having anyone attend.
- [Salesforce has meeting rooms that they allow nonprofits to book](https://www.salesforce.com/company/ohana-floors-chicago/), I submitted a request.

**Comment by timsaucer** [comment]
Dec is usually a bit busy for me but I might be able to make it out that week.

**Comment by alamb** [comment]
I made a new page just for the Dec 18 event: https://github.com/apache/datafusion/discussions/12894

Event signup is https://lu.ma/eq5myc5i

**Comment by alamb** [comment]
Here are the slides and recordings from Dec 18, 2024 (I also plan to write a blot post about this)


* [VIDEO](https://youtu.be/XxEtOf-MzNA?si=CHSqzeRr1Sh7ZpD-) / [SLIDES](https://docs.google.com/presentation/d/1BWuVyGzF1_iQRcmZDfoDg4V5kVlefw1fDGRt7TqxYsU/edit): ​Building a real-time data lake with DataFusion, Adrian Garcia Badaracco @adriangb  - Pydantic, Founding Engineer
* ​[VIDEO](https://youtu.be/CsqWOxFWK9w?si=AsCUDxDWbKtBoA_F) / [SLIDES](https://timsaucer.com/data/Practical_Robotics_with_DataFusion.pdf): Practical data science in robotics using DataFusion, Tim Saucer @timsaucer - May Mobility, Director of Simulation & Infrastructure
*  [VIDEO](https://youtu.be/woqXLE25gMc?si=1Irx1i70qjmujBj3) / [SLIDES](https://github.com/user-attachments/files/18214371/datafusion-cache.pdf): Practical Disaggregated Cache for DataFusion - Xiangpeng Hao @XiangpengHao , PhD Student, UW Madison
* [VIDEO](https://youtu.be/-1D25zAfB00?si=QY5EB5x9kKXGDVus) (no sound 😢 ) / [SLIDES](https://docs.google.com/presentation/d/1AiI5r8LtnbDkceBW1NSlqBmD07xxq1JJWQFkRZ35Ef8/edit#slide=id.g2e79518841d_0_95): Building InfluxDB 3.0 with the FDAP stack: Andrew Lamb, @alamb InfluxData, Staff Engineer, DataFusion, PMC chair

Playlist: https://www.youtube.com/playlist?list=PLrhIfEjaw9in_Gcyl0KBFzMdi3XEhETIT
